products/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
+
+from products.models import SubCategory
 
 # Create your views here.
+
+class SubCategoryView(View):
+    def get(self, request, category_id):  
+        
+        subcategories = SubCategory.objects.filter(category_id=category_id)
+        result        = []
+        
+        if not subcategories:
+            return JsonResponse({'message':'Subcategory does not exist.'}, status=400)
+        
+        for subcategory in subcategories:
+            result.append({
+                'id'         : subcategory.id,
+                'name'       : subcategory.name,
+                'image_url'  : subcategory.image_url,
+                'category_id': category_id,
+            })
+        
+        return JsonResponse({'message':result}, status=200)
```

Comment:
message는 정말 해당 응답이 어떤 응답인지 말해 줄 때 많이 사용하고, 데이터를 보낼 때는 'data' 또는 'result'를 많이 사용합니다.
```suggestion
        return JsonResponse({'result':result}, status=200)
```

Reply:
앗 넵 수정하겠습니다!!